### PR TITLE
EAR-1244 Update Summary and Thank You page wording

### PR DESCRIPTION
**Pull request by jchapman68**
### What is the context of this PR?
Update wording to show respondants can save and print to pdf

### How to review 
Check wording on surveys match the ACs and supplied document

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

File: app/themes/covid/templates/thank-you.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <h1 class="u-fs-l" data-qa="submission-successful-title">{{ _("Submission succes
     {% if view_submission_url %}
     <p class="u-mb-m@s u-fs-r" data-qa="view-submission"><a href={{view_submission_url}} data-ga="click"
             data-ga-category="view-submission Covid" data-ga-action="Goto covid view-submission page"
-            data-ga-label="Goto covid survey view-submission page">{{ _("View and print a copy of your answers") }}</a>
+            data-ga-label="Goto covid survey view-submission page">{{ _("VView, save or print a copy of your answers") }}</a>
```
Comment (tjamesmac) [inline app/themes/covid/templates/thank-you.html]:
```suggestion
            data-ga-label="Goto covid survey view-submission page">{{ _("View, save or print a copy of your answers") }}</a>
```
Spelling mistake here

Comment (tbaggaley) [inline app/themes/covid/templates/thank-you.html]:
```suggestion
            data-ga-label="Goto covid survey view-submission page">{{ _("View, save or print a copy of your answers") }}</a>
```

**Review by tbaggaley — CHANGES_REQUESTED** [review]

File: app/themes/covid/templates/thank-you.html (hunk 1 above)
Comment (tbaggaley) [reply app/themes/covid/templates/thank-you.html]: Jinx!!

Comment (tjamesmac) [reply app/themes/covid/templates/thank-you.html]: Damn

Comment (tjamesmac) [reply app/themes/covid/templates/thank-you.html]: Got me

**Review by tbaggaley — APPROVED** [review]

**Review by MattElcock — APPROVED** [review]
Hi Josh, nice work here. Thanks for doing this!

**Pull request merged**